code/modules/spells/spell_types/vampire_powers.dm

```diff
@@ -351,7 +355,7 @@
 	user.regenerate_organs()
 	user.Stun(60) //Can just circumvent by becoming a bat, need to find a way around this
 
- /obj/effect/proc_holder/spell/self/summon_coat
+ /obj/effect/proc_holder/spell/self/summon_coat //Becomes a basic wizard spell which you need wizard clothes to do, possibly tied to making the coat itself a subtype of something?
```

Comment:
You need to set clothes_req to FALSE for it to be cast without robes.

Reply:
And the icon for it is not showing up, only using the default one wierdly.

Reply:
Oh, so the summoned coat has a different sprite than the standard one?